src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -79,7 +82,9 @@ public void sendGameOveNotification(String gameId){
 
     @MessageMapping("/playerUpdate")
     public GuessResult sendGuessResult(GuessResult result){
-        messagingTemplate.convertAndSend("/game1/playerUpdate/" + result.getGameId(), result);
+        StringBuilder playerUpdateIdString = new StringBuilder("/game1/playerUpdate/")
+                .append(result.getGameId());
+        messagingTemplate.convertAndSend(playerUpdateIdString.toString(), result);
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/

Reply:
StringBuilder is a good idea, if you are performing multiple String concatenations.  Good catch!